src/child/index.js

```diff
@@ -30,6 +30,12 @@ currentWindowService.ready(() => {
     const windowStateService = new WindowStateService(currentWindow, store);
     windowStateService.start();
 
+    currentWindow.addEventListener('bounds-changed', (e) => {
```

Comment:
I think this should be added to the `WindowStateService`, along with the other window event listeners,